packages/components/src/horizontal-bar-list/style.scss

```diff
@@ -28,17 +28,32 @@
 		&:hover,
 		&:focus,
 		&:focus-within {
-			background-color: var(--studio-gray-0);
+			background-color: var(--theme-highlight-color);
+			color: var(--studio-white);
 
-			.horizontal-bar-list-item-bar {
-				&::before {
-					background-color: rgba(var(--color-primary-rgb), 40%);
-				}
+			svg {
+				fill: var(--studio-white);
 			}
 
 			.horizontal-bar-list--hover-action {
 				display: block;
 			}
+
+			.horizontal-bar-list-label {
+				color: var(--studio-white);
+			}
+
+		}
+	}
+
+	.horizontal-bar-list-group {
+		.horizontal-bar-list-item {
+			&:hover,
+			&:focus,
+			&:focus-within {
+				margin-left: $stats-child-hover-indentation;
```

Comment:
I didn't implement any indentation for nested lists. The hover is indented, but the actual rows are not. I did it to keep the same reference point for the bar's length and relation.
The downside of this approach is that it's harder to see which items are expanded.
Which version do you think is better @keoshi @jeffgolenski?

If an indentation is needed for the normal view (not only hover), this margin has to be moved 3 lines up. That should fix the issue.

Reply:
Regarding the indentation, there could be a background-color transition issue with setting the margin and padding on the `list item` directly, which would not appear with putting the padding on the `list`.


https://user-images.githubusercontent.com/6869813/209045953-50ad87cc-4d7e-4c11-bee8-a6a9ffd31c61.mov

Reply:
That's a good point. I can't put the margin on an element other than the list, which is also a list item, because the highlight background won't take the whole width of the parent items.

This made me think that only one level of indentation for hover and no indentation for regular (not hover) view looks very strange for elements with more than two levels of nesting (e.g. Referrers for some pages).
I added margin to a group which added indentation to all of the children and applied it to all child groups independent of the indentation level.

If this doesn't look right, we can revert the latest commit and fix the transition issue.

Reply:
The PR description has updated images of nested indentation (implemented in commit [e8234a0](https://github.com/Automattic/wp-calypso/pull/71437/commits/e8234a0fa129ad7816acacc82e8e13ba627aec20)). Before the change, it looked like this:

![Screenshot 2022-12-22 at 2 22 21 PM](https://user-images.githubusercontent.com/112354940/209034817-a3ead084-463e-49d1-98ab-645f66f2339b.png)
![Screenshot 2022-12-22 at 2 22 40 PM](https://user-images.githubusercontent.com/112354940/209034812-70f66bde-9c86-4957-a8cd-cb5edc0e5e35.png)